common/views.py

```diff
@@ -1,10 +1,12 @@
 from django.contrib.auth.views import LoginView, LogoutView
 from django.contrib.auth import login, authenticate, logout
 from django.contrib.auth.forms import UserCreationForm
+from common.forms import RegisterForm
```

Comment:
placements are wrong,
sperate the project imports from django's builtin
and should be in order